index.html

```diff
@@ -1088,6 +1089,257 @@ <h4>Nonce</h4>
       </section>
     </section>
 
+    <section>
+      <h2>The BBS+ Bound Signature Suite 2020</h2>
+
+      <p>
+The BBS+ Bound <a>signature suite</a> 2020 adds a mechanism for recipient
+binding, and is otherwise identical, to the BBS+ <a>signature suite</a>
+2020. It MUST be used in conjunction with the signing and verification
+algorithms in the Linked Data Proofs [[LD-PROOFS]] specification. The suite
+consists of the following algorithms:
+      </p>
+      <table class="simple">
+        <thead>
+        <th>Parameter</th>
+        <th>Value</th>
+        <th>Specification</th>
+        </thead>
+        <tbody>
+        <tr>
+          <td><a>canonicalization algorithm</a></td>
+          <td>https://w3id.org/security#URDNA2015</td>
+          <td>[[RDF-DATASET-NORMALIZATION]]</td>
+        </tr>
+        <tr>
+          <td><a>statement digest algorithm</a></td>
+          <td>Blake2b</td>
+          <td>[[BLAKE2]]</td>
+        </tr>
+        <tr>
+          <td><a>signature algorithm</a></td>
+          <td>BBS+ Signature</td>
+          <td>[[BBS]]</td>
+        </tr>
+        <tr>
+          <td><a>curve name</a></td>
+          <td>BLS12-381</td>
+          <td>[[PAIRING-FRIENDLY-CURVES]]</td>
+        </tr>
+        </tbody>
+      </table>
+
+      <section>
+        <h3>Modification to Algorithms</h3>
+
+        <section>
+          <h4>Create Verify Data Algorithm</h4>
+
+          <p>
+The create verify data algorithm defined below is identical to the create verify
+data algorithm defined for the BBS+ <a>signature suite</a> 2020, with the
+addition of steps to include recipient binding.
```

Comment:
Future thought, I think we should maybe add a definition for "recipient" (if thats the word we want to use) in this document